clients/drcachesim/tests/drcachesim_unit_tests.cpp

```diff
@@ -504,12 +505,55 @@ unit_test_cache_line_size()
     }
 }
 
+// Verify that illegal cache configurations are rejected.
+void
+unit_test_cache_bad_configs()
+{
+    // Safe values we aren't testing.
+    const int safe_assoc = 1;
```

Comment:
I guess the style guide would suggest `static constexpr int SAFE_ASSOC = 1;` and similar below?

Reply:
Done.